docs/make_work.jl

```diff
@@ -83,19 +83,24 @@ function setup_experimental_package(Oscar::Module, package_name::String)
 
   # Set symlink inside docs/src/experimental
   symlink_link = joinpath(Oscar.oscardir, "docs/src/Experimental", package_name)
-  symlink_target = joinpath("../../../experimental", package_name, "docs/src")
+  symlink_target = joinpath(Oscar.oscardir, "experimental", package_name, "docs/src")
```

Comment:
Weird, I was just told that symlink targets are always relative paths, but apparently in julia they aren't. @benlorenz

Reply:
I just made it absolute because then the `ispath`-check is easier. And it appears to work like this...

Reply:
Well, the testsuite runs and it works on my machine, and I also think that the code is more readable like this, so thank you. I probably just misunderstood something.